packages/charts/src/Charts/SeverityLine/SeverityLine.js

```diff
@@ -0,0 +1,73 @@
+import './severity-line.scss';
+
+import React, { useEffect, useState } from 'react';
+import { Tooltip, TooltipPosition } from '@patternfly/react-core/dist/js/components/Tooltip/index';
```

Comment:
No need to import like this, the app will decide what to use later on, you can use `import { Tooltip, TooltipPosition } from '@patternfly/react-core';`